Defs/Ammo/Grenade/20x42mmGrenade.xml

```diff
@@ -15,6 +15,7 @@
     <label>20x42mm Grenades</label>
     <ammoTypes>
     	<Ammo_20x42mmGrenade_HE>Bullet_20x42mmGrenade_HE</Ammo_20x42mmGrenade_HE>
+		<Ammo_20x42mmGrenade_HEAB>Bullet_20x42mmGrenade_HEAB</Ammo_20x42mmGrenade_HEAB>
```

Comment:
As a general comment, the defName structure and labels for the airburst rounds should be consistent across all the different ammos that have an airburst round.

While the current format (below) isn't as tidy, it is a bit more readily adaptable for potential non-HE airburst shells that come along in the future.
`<Ammo_30x29mmGrenade_HE_TFuzed>Bullet_30x29mmGrenade_HE_TFuzed</Ammo_30x29mmGrenade_HE_TFuzed>`

In any case, we should probably pick one format and stick to it. If we care to tidy up the current one we're using, maybe: `Bullet_30x29mmGrenade_HE_AB`? The release for 1.4 would be a good time to do it, since changing defNames is somewhat save-unsafe.